doc/modules/ROOT/pages/index.adoc

```diff
@@ -1,4 +1,4 @@
-= The Neo4j Graph Data Science Client Manual v{docs-version}
+= Introduction
```

Comment:
was this maybe also for seo? / should we add keywords?

Reply:
We don't _have_ to, but I agree it's clearer to keep a reference to the product in the title. We can still use a different name for the menu item. WDYT?

Reply:
i like your suggestion to keep the title but simplify the menu item 👍